.travis.yml

```diff
@@ -4,14 +4,22 @@ language:
   - ruby
 node_js:
   - 6
+cache:
+  directories:
+    - ~/.npm
+    - ~/.cache
```

Comment:
This guide seems complete. You can switch Node.js version that we're using a few lines above. Try:
```
node_js:
  - node
```
instead of `- 6`. It should install the latest stable Node.js (and NPM), perhaps v10. Then you should have access to `npm ci`.